src/tasks.js

```diff
@@ -191,11 +259,77 @@ const sections = [
       tasks.MANAGEMENT_SYSTEM,
       tasks.FIRE_PREVENTION_PLAN,
       tasks.CONFIRM_CONFIDENTIALLY,
+      tasks.INVOICING_DETAILS
+    ]
+  },
+  {
+    id: 'send-and-pay-section',
+    label: 'Apply',
+    tasks: [
+      tasks.SUBMIT_PAY
+    ]
+  }
+]
+
+const bespoke = [
+  {
+    id: 'get-a-save-link-section',
+    label: 'Get a save link',
+    tasks: [
+      tasks.SAVE_AND_RETURN_EMAIL
+    ]
+  },
+  {
+    id: 'about-the-application-section',
+    label: 'About the application',
+    tasks: [
+      tasks.PRE_APPLICATION,
+      tasks.PERMIT_HOLDER_DETAILS,
+      tasks.CONFIRM_CONFIDENTIALLY,
       tasks.INVOICING_DETAILS,
-      // TODO: EWC: Remove once this is in bespoke
+      tasks.NEED_TO_CONSULT
+    ]
+  },
+  {
+    id: 'operations-section',
+    label: 'Operations',
+    tasks: [
+      tasks.BASIC_OPERATION_DETAILS,
+      tasks.NON_TECHNICAL_SUMMARY,
+      tasks.SITE_NAME_LOCATION,
+      tasks.SITE_CONDITION_REPORT,
+      tasks.SITE_PLAN
+    ]
+  },
+  {
+    id: 'activities-section',
+    label: 'Activities',
+    tasks: [
+      tasks.RECOVERY_AND_DISPOSAL_CODES,
       tasks.WASTE_TYPES_LIST
```

Comment:
Now that this is here, it should be possible to clear the ruleSetId for this task

Reply:
Cleared